charts/newrelic-k8s-metrics-adapter/values.yaml

```diff
@@ -43,17 +51,25 @@ config:
   #   removeClusterFilter: false
 
 image:
```

Comment:
Rather than specifying comments for each of the image items it might be enough to cover the whole image object to avoid some noise in the readme:

```suggestion
# image -- Registry, repository, tag, and pull policy for the container image.
# @default -- See `values.yaml`.
image:
```

If I remember correctly, if you specify the `@default` and skip comments for the items inside, `helm-docs` will not add them to the table.